ui/mainwindow.ui

```diff
@@ -120,6 +120,14 @@
    </property>
   </action>
  </widget>
+ <customwidgets>
+  <customwidget>
+   <class>InstrumentView</class>
+   <extends>QWidget</extends>
+   <header>nexus_constructor/InstrumentView.h</header>
```

Comment:
Could you stick something in the WIKI about how this works? cheers

Reply:
I've tried to do that here https://github.com/ess-dmsc/nexus-constructor/wiki/Qt-Designer-Guide#using-custom-widgets